docs/UserGuide.adoc

```diff
@@ -679,54 +719,142 @@ Examples:
 // tag::statisticsmode[]
 === Statistics Mode
 
-In Statistics Mode, you can obtain statistics related to your restaurant, such as daily revenue and the popularity of dishes.
+In Statistics Mode, you can obtain statistics related to your restaurant, such as the popularity of dishes and the
+revenue of a specified year, month or day.
+
+The user interface for Statistics Mode is shown in the image below.
 
 image::SMUi.png[width="790"]
 
-[[Revenue]]
-==== Getting the total revenue : `revenue`
+The function of the side and main panels are as follows:
+
+* *Item Panel*: This panel shows you the list of menu items sorted by its popularity. This is meant to help you decide
+what ingredients to order to cater to the demands of the popular dishes.
+* *Revenue Panel*: This panel displays the revenue from the past 30 years, months and days for you to study the revenue trends.
```

Comment:
years, months or days, depending on the view that you have selected. This allows you to study trends in your restaurant's revenue.